src/components/state-details.js

```diff
@@ -731,6 +738,12 @@ export default function StateDetailsPage({ location, data }) {
                 <strong>{solarPanelsCountStr} Megawatts</strong> of solar.
               </p>
 
+              <p className="mt-6">
+                Since the average wind turbine providees 2.75 MW of peak
```

Comment:
@SuragNuthulapaty - I'm impressed how fast you responded, but make sure to check your spelling 😆 ("providees" -> "provides")